src/cloud-code/Parse.Cloud.js

```diff
@@ -677,6 +677,42 @@ ParseCloud.onLiveQueryEvent = function (handler) {
   triggers.addLiveQueryEventHandler(handler, Parse.applicationId);
 };
 
+/**
+ * Registers a before live query subscription function.
+ *
+ * **Available in Cloud Code only.**
+ *
+ * If you want to use beforeUnsubscribe for a predefined class in the Parse JavaScript SDK (e.g. {@link Parse.User}), you should pass the class itself and not the String for arg1.
+ * ```
+ * Parse.Cloud.beforeUnsubscribe('MyCustomClass', (request) => {
+ *   // code here
+ * }, (request) => {
+ *   // validation code here
+ * });
+ *
+ * Parse.Cloud.beforeUnsubscribe(Parse.User, (request) => {
+ *   // code here
+ * }, { ...validationObject });
+ *```
+ *
+ * @method beforeUnsubscribe
+ * @name Parse.Cloud.beforeUnsubscribe
+ * @param {(String|Parse.Object)} arg1 The Parse.Object subclass to register the before subscription function for. This can instead be a String that is the className of the subclass.
+ * @param {Function} func The function to run before a subscription. This function can be async and should take one parameter, a {@link Parse.Cloud.TriggerRequest}.
+ * @param {(Object|Function)} validator An optional function to help validating cloud code. This function can be an async function and should take one parameter a {@link Parse.Cloud.TriggerRequest}, or a {@link Parse.Cloud.ValidatorObject}.
+ */
+ParseCloud.beforeUnsubscribe = function (parseClass, handler, validationHandler) {
+  validateValidator(validationHandler);
+  var className = getClassName(parseClass);
```

Comment:
Prefer `const`

Reply:
Interesting, shouldn't lint pick that up? I guess we don't have that rule enabled...

Reply:
There is a few other `var` in that file too. That one was copied from another trigger.

Reply:
Well, we can either use `const` here if it is a constant, or we enable the lint rule for `const` in a separate PR and it will probably auto-fix.

Reply:
Discussed [here](https://github.com/parse-community/parse-server/issues/7260)